dask/array/tests/test_creation.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from numpy.testing import assert_array_almost_equal
+
+from dask.array import creation
+
+
+def test_linspace():
+    darr = creation.linspace(6, 49, 5)
+    nparr = np.linspace(6, 49)
+    assert_array_almost_equal(np.array(darr), nparr)
```

Comment:
I'll re-define it in this module (since I cannot import it). Then add a `testing_utils.py` or something in another PR that pulls all the redundant definitions out of all the testing modules.